src/Popup/PopupInsert/PopupActionGroup/index.js

```diff
@@ -18,8 +18,19 @@ class PopupActionGroup extends Component {
     }
   }
 
-  onHover (hover) {
-    this.setState({ showFlyout: hover })
+  onHover (hover, delay = false) {
+    if (delay) {
+      // allow a slight delay to account for mousing over scroll bars
+      // when moving to flyout on certain browsers/OS
+      this.timeout = setTimeout(() => this.setState({ showFlyout: hover }), 50)
+    } else {
+      if (this.timeout) {
```

Comment:
Would it be better to do this clear-down of `this.timeout` irrespective of the `delay` argument before setting up the timeout or direct setting? It looks wrong in that if you call `onHover(hover, true)` to set up the timeout, and then it gets called again _before_ the timeout expires then we'll leak the original `setTimeout` return. Additionally it seems a little odd/dangerous that calling it once with `true` for delay and then calling it again with `false` for delay is the expected path.